client.go

```diff
@@ -516,9 +571,9 @@ func (c *Client) CertificateRemove(fingerprint string) error {
 	return ParseError(raw)
 }
 
-func (c *Client) Init(name string) (*Response, error) {
+func (c *Client) Init(name string, image string) (*Response, error) {
 
-	source := shared.Jmap{"type": "remote", "url": "https+lxc-images://images.linuxcontainers.org", "name": "lxc-images/ubuntu/trusty/amd64"}
+	source := shared.Jmap{"type": "local", "name": image}
```

Comment:
Reminds me I need to update the REST API document with the various input that's allowed as a container source.

I expect it to be something like:

    {
         "type": "image",
         "fingerprint": "SHA-256"
    }

or

    {
         "type": "image",
         "alias": "ubuntu"
    }

or

    {
         "type": "image",
         "properties": {
              "os": "ubuntu",
              "release": "14.04"
         }
    }

or just no source if defining a container based on an existing rootfs (as will be the case for nova-compute-lxd).